app/models/suggestion.py

```diff
@@ -0,0 +1,119 @@
+from .. import db
+from datetime import datetime
+import pytz
+
+
+class Suggestion(db.Model):
+    """
+    Association between a resource and potential suggestions for it
+    """
+    __tablename__ = 'suggestions'
+    id = db.Column(db.Integer, primary_key=True)
+    # resource_id is -1 for insertion
+    resource_id = db.Column(db.Integer, db.ForeignKey('resources.id'))
+    suggestion_text = db.Column(db.String(250))
+    # 0 stands for read, 1 stands for unread
+    read = db.Column(db.Boolean, default=False)
+    timestamp = db.Column(db.DateTime)
+    contact_name = db.Column(db.String(64))
+    contact_email = db.Column(db.String(64))
+    contact_number = db.Column(db.String(64))
+
+    def __repr__(self):
+        return '%s: %s' % (self.suggestion_type, self.resource_id)
+
+    @staticmethod
+    def generate_fake_inserts(count=20):
+        """Generate a number of fake insert suggestions."""
+        from sqlalchemy.exc import IntegrityError
+        from faker import Faker
+
+        fake = Faker()
+
+        num_words = 10
+        for i in range(count):
+            s_text = fake.sentence(nb_words=num_words)
+            s_read = 1
+            s_timestamp = datetime.now(pytz.timezone('US/Eastern'))
+            s_contact_name = fake.word()
+            s_contact_email = fake.word() + "@" + fake.word() + ".com"
+            s_contact_number = "123-456-7890"
+            s_insert = Suggestion(resource_id=-1, suggestion_text=s_text,
```

Comment:
I think `resource_id` should just be omitted - then it'll create a `Suggestion` without a relationship to a `Resource`.